app/models/credit_note.rb

```diff
@@ -106,7 +106,7 @@ def add_on_items
   end
 
   def should_sync_credit_note?
-    finalized? && customer.integration_customers.accounting_kind.any? { |c| c.integration.sync_credit_notes }
+    finalized? & customer.accounting_customer&.integration&.sync_credit_notes
```

Comment:
updated to `bitwise and` because `logical and` returns value if it exist instead of true/false
So in our case if second half returns nil, 
&& - returns nil for the whole operation
& - returns false for the whole operation,
no matter what first argument was

Reply:
would `&& customer.accounting_customer&.integration&.sync_credit_notes.present?` work here? I have nothing against "bitwise and" but without you're comment I'm not sure I'd have even noticed ^^